atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -92,6 +98,24 @@ private TransactionManagers() {
     }
 
     /**
+     * Create a {@link SerializableTransactionManager} backed by an
+     * {@link com.palantir.atlasdb.keyvalue.impl.InMemoryKeyValueService}.  This should be used for testing
+     * purposes only.
+     */
+    public static SerializableTransactionManager createInMemory(AtlasSchema schema, AtlasSchema... otherSchemas) {
+        return create(ImmutableAtlasDbConfig.builder().keyValueService(new InMemoryAtlasDbConfig()).build(),
```

Comment:
this line looks so weird... I guess it's because `.keyValueService(arg)` sets a `KeyValueServiceConfig` and not a `KeyValueService`.